README.md

```diff
@@ -1,140 +1,143 @@
-* [Nunjucks](https://mozilla.github.io/nunjucks/) __Templates__ (für den Prototyp, könnten aber auch für JavaScript-Komponenten verwendet werden)
-* ein konfigurierbarer [Modernizr](https://modernizr.com) (weil es geht)
-* __Bilder__ werden minifiziert
-* __SVGs__ werden kombiniert und ins HTML gebracht (für Icons)
-* [Bootstrap](http://getbootstrap.com) und [Google Material Icons](https://material.io/icons/) werden als externe Komponenten (via npm) eingebunden
-* [BrowserSync](https://www.browsersync.io) für __Live-Reload__ und zum Test auf verschiedenen Geräten
+* [Yarn](https://yarnpkg.com) as a __package manager__
+* [Gulp](http://gulpjs.com) as a __task runner__
+* [Sass](http://sass-lang.com) and [PostCSS](http://postcss.org) for __CSS__ (with [Autoprefixer](http://autoprefixer.github.io), [cssnano](http://cssnano.co) and Bimmelbam)
+* ES6 with [Babel](http://babeljs.io) and [Browserify](http://browserify.org) for beautiful __JavaScript__
+* [Nunjucks](https://mozilla.github.io/nunjucks/) __Templates__ (for the prototype, but could also be used for JavaScript components)
+* a customizable [Modernizr](https://modernizr.com) (because it works)
+* __Images__ be minified
+* __SVGs__ are combined and injected into the HTML (for icons)
+* [Bootstrap](http://getbootstrap.com) and [Google Material Icons](https://material.io/icons/) are included as external components (via npm)
+* [BrowserSync](https://www.browsersync.io) for __Live-Reload__ and for testing on different devices
 
-Nicht enthalten, aber sehr sinnvoll:
+Not included, but very useful:
 
-* __Linting__, damit Code den Anforderungen entspricht.
-* __Testing__, weil Menschen ständig Fehler machen.
-* __Continuous integration (CI)__, um das Projekt nicht immer von Hand zusammenbauen zu müssen.
-* __Deployment__ und sonstiges Bimmelbam
+* __Linting__, so that code meets the requirements.
+* __Testing__, because people are constantly making mistakes.
+* __Continuous integration (CI)__, not always having to assemble the project by hand.
+* __Deployment__ and other Bimmelbam
 
 ## Setup
 
-1. __Node__ (>= 6.9) installieren, falls noch nicht vorhanden. Kann als Paket direkt von der Website runtergeladen werden: [https://nodejs.org](https://nodejs.org)
-2. __Yarn__ installieren, falls noch nicht vorhanden ([Anleitung](https://yarnpkg.com/en/docs/install)).
-3. Im Terminal in unser Verzeichnis wechseln und mittels Yarn die benötigten Pakete holen.  
-_Achtung, es kommen mehrere hundert Megabyte an Daten durch die Leitung!_  
+1. Install __Node__ (>= 6.9), if not already available. Can be downloaded as a package directly from the website: [https://nodejs.org](https://nodejs.org)
+2. Install __Yarn__, if not already available ([Manual](https://yarnpkg.com/en/docs/install)).
+3. Change to our directory in the terminal andand fetch the required packages packages.
+
+
+_Attention, several hundred megabytes of data are coming through the line!_  
```

Comment:
Sentence contains 2x »and« and 2x »packages«:
»3. Change to our directory in the terminal and~~and~~ fetch the required ~~packages~~ packages.«

Also, you inserted several new lines. However, what we need is **two spaces** behind »packages.«, followed by »Attention, several…« in new line.